test/index.js

```diff
@@ -19,6 +19,7 @@ describe('Klaviyo', function () {
       listId: 'baVTu8',
       sendAnonymous: true,
       enforceEmail: false,
+      useSegmentSpec: false
```

Comment:
@ladanazita but following your example with Optimizely, I'll make the default here to `true` and flip it to `false` for existing user.